src/tests/artists.js

```diff
@@ -0,0 +1,42 @@
+const actions = {
+	create: require("../actions/artists/create"),
+	delete: require("../actions/artists/delete"),
+	findAll: require("../actions/artists/find-all"),
+	findOne: require("../actions/artists/find-one"),
+	update: require("../actions/artists/update"),
+};
+
+describe("Artist actions", () => {
+	it("returns a list of artists", () => {
+		actions.findAll().then(response => {
+			expect(response.ok).toBe(true);
+			expect(response.artists[0].get("name")).toBe("Micro Jackson");
+		});
+	});
+
+	it("returns a single artist", () => {
+		actions.findOne(1).then(response => {
+			expect(response.ok).toBe(true);
+			expect(response.artist.name).toBe("Micro Jackson");
+		});
+	});
+
+	it("creates a new artist", () => {
```

Comment:
@BlackChaosNL Perhaps you know this better than I do right now, but during tests, this one fails with the following response:

```
{
    ok: false,
    message: 'This artist already exists.'
}
```